src/snapshot.ts

```diff
@@ -185,7 +188,10 @@ export class Snapshot {
    *   const apiResponse = data[0];
    * });
    */
-  delete(callback) {
+  delete(): Promise<google.protobuf.Empty>;
```

Comment:
Why is this needed?

Reply:
function signature for information of overloading!